Mathlib/Algebra/GroupWithZero/Basic.lean

```diff
@@ -407,7 +407,7 @@ lemma zero_zpow_eq_one₀ {n : ℤ} : (0 : G₀) ^ n = 1 ↔ n = 0 := by
 
 lemma zpow_add_one₀ (ha : a ≠ 0) : ∀ n : ℤ, a ^ (n + 1) = a ^ n * a
   | (n : ℕ) => by simp only [← Int.ofNat_succ, zpow_natCast, pow_succ]
-  | .negSucc 0 => by simp [ha]
+  | -1 => by simp [ha]
```

Comment:
This is not essential. I'm just trying to make `negSucc` less visible / more quickly dealt with by `simp`.